readthedocs/core/static-src/core/js/doc-embed/sponsorship.js

```diff
@@ -78,8 +86,9 @@ function create_ad_placement() {
         return $('<div />')
             .attr("id", "rtd-sidebar")
             .attr("data-ea-publisher", "readthedocs")
-            .attr("data-ea-type", "readthedocs-sidebar")
+            .attr("data-ea-type", ad_type)
             .attr("data-ea-manual", "true")
+            .attr("data-ea-style", style_name)
             .addClass(class_name)
```

Comment:
Probably a dumb question but, does this work correctly when `ad_type` and `style_name` are undefined/none variables? Same question applies for `class_name`.